cvat-core/src/server-proxy.ts

```diff
@@ -1841,6 +1841,17 @@ async function installedApps() {
     }
 }
 
+async function getApiSchema() {
+    const { backendAPI } = config;
+
+    try {
+        const response = await Axios.get(`${backendAPI}/schema/?scheme=json`);
```

Comment:
Now this request returns full schema. I dont really remember what we decided about this in our team discussion, but do we really need that? Looks like a lot of information is useless and can be filtered

Reply:
We decided to use the whole schema without filtration. I believe it is better to request schema one time and then use data from the state. For example, [here](https://github.dev/opencv/cvat/blob/2b33548411b39a8e8dd60a2a72fce4ca93145adc/cvat-core/src/api-implementation.ts#L510) we need information about components. Probably we can replace /server/about endpoint with schema also.